translations/es.json

```diff
@@ -0,0 +1,5 @@
+{
+  "content.lessons.genesis.blockExplorer": "explorador de bloques",
+  "content.lessons.genesis.blockExplorerTooltipContent": "",
+  "content.lessons.genesis.firstParagraph": ""
```

Comment:
Translators can easily scan the file for blank strings to see where translations are needed.

Reply:
Tools like [Crowdin](https://crowdin.com/) could also be used with this setup.